src/modules/ie-objects/elasticsearch/queryBuilder.ts

```diff
@@ -78,6 +73,35 @@ export class QueryBuilder {
 			const max = Math.max(0, this.config.MAX_COUNT_SEARCH_RESULTS - limit);
 			queryObject.from = clamp(offset, 0, max);
 
+			if (
+				searchRequest.filters.some((filter: SearchFilter) =>
+					[
+						SearchFilterField.CONSULTABLE_REMOTE,
+						SearchFilterField.CONSULTABLE_MEDIA,
+					].includes(filter.field)
+				)
+			) {
+				inputInfo = {
+					...inputInfo,
+					isConsultableRemote: searchRequest.filters.some(
+						(filter: SearchFilter) =>
+							filter.field === SearchFilterField.CONSULTABLE_REMOTE
+					),
+					isConsultableMedia: searchRequest.filters.some(
+						(filter: SearchFilter) =>
+							filter.field === SearchFilterField.CONSULTABLE_MEDIA
+					),
+				};
+
+				searchRequest.filters = searchRequest.filters.filter(
```

Comment:
add comment:
// Remove CONSULTABLE_REMOTE and CONSULTABLE_MEDIA filter entries from the filter list, since they have been handled above and should not be handled by the standard field filtering logic.